open-metadata-implementation/access-services/data-engine/data-engine-server/src/main/java/org/odpi/openmetadata/accessservices/dataengine/server/handlers/ProcessHandler.java

```diff
@@ -325,7 +328,8 @@ public void updateProcessStatus(String userId, String guid, InstanceStatus insta
             return new HashSet<>();
         }
 
-        return entities.parallelStream().map(InstanceHeader::getGUID).collect(Collectors.toSet());
+        return entities.parallelStream().filter(entityDetail -> entityDetail.getType().getTypeDefName().equalsIgnoreCase(portTypeName))
```

Comment:
Not sure if I understand why we need portType as a parameter here from process for extra checking?

Reply:
Because I want to retrieve only the port implementations/port aliases for a certain process. repositoryHandler.getEntitiesForRelationshipType will retrieve all the entitites for a ProcessPort relationship that have the first end the processGUID, and I need the result filtered based on the actual type